keps/sig-apps/2307-job-tracking-without-lingering-pods/README.md

```diff
@@ -666,7 +666,7 @@ Yes, see [Deprecation](#deprecation) for the full plan.
       - Before 1.26: Observe the behavior in pods.
       - After 1.26: Based on metric `job_terminated_pod_tracking_finalizer`
     - Mitigations:
-      Before 1.26, disable `JobTrackingWithFinalizers`.
+      - Update to latest patch version of supported Kubernetes versions. [119833](https://github.com/kubernetes/kubernetes/issues/119833) was backported in patch versions from 1.24 and up.
```

Comment:
the issues are listed below, under diagnostics.

Yes, remove the Finished jobs section.

Reply:
Updated.

Reply:
I think I wasn't clear. Move the link to the issue below (line 675)

Reply:
ah. done.